techlibs/ice40/ice40_unlut.cc

```diff
@@ -74,7 +74,7 @@ static void run_ice40_unlut(Module *module)
 }
 
 struct Ice40UnlutPass : public Pass {
-	Ice40UnlutPass() : Pass("ice40_unlut", "iCE40: perform simple optimizations") { }
+	Ice40UnlutPass() : Pass("ice40_unlut", "iCE40: transform SBLUT4 cells to $lut cells") { }
```

Comment:
Really small nitpick: should be `SB_LUT4`

Reply:
I just pushed another commit to fix it. Thanks!